### PR TITLE
Make collaborates edditable

### LINKED ISSUE #100: IntegrityError for redundant collaborator emails
I get an `IntegrityError` if I add a collaborator mail address twice during the idea creation.

Although I get an error, the project is created and listed in the idea space.

### PR DESCRIPTION
**Pull request by slomo**
Also fix #100

### CODE REVIEW

File: apps/ideas/views.py [hunk 1]
```diff
@@ -266,7 +267,8 @@ class ProposalEditView(
         forms.ImpactSectionForm,
         forms.SelectionCriteriaSectionForm,
         forms.FinanceAndDurationSectionForm,
-        forms.CommunitySectionForm
+        forms.CommunitySectionEditForm,
+        forms.CommunitySectionEditForm,
```
Comment (fuzzylogic2000) [inline apps/ideas/views.py]: 2 times?

File: apps/ideas/forms.py [hunk 2]
```diff
@@ -170,3 +189,134 @@ class Meta:
...
+
+        if collaborator_count > 5:
+            raise ValidationError(_('Maximum 5 collaborators allowed'))
+
+    def save(self, commit=True):
+        """
+        Deletes invites and collaborators and adds new invites of instance.
+
+        There is a little hack here, it uses the idea creator and not the
+        current user as creator for the invites. There for no user needs to
```
Comment (fuzzylogic2000) [inline apps/ideas/forms.py:370]: I think that's fine as the whole idea is also created by that creator and the collaborators are also only a model field after all.

**Review by fuzzylogic2000 — COMMENTED** [review]
Looks fine to me! But will still have to check if it also works.

**Review by fuzzylogic2000 — COMMENTED** [review]
Does not work locally - I can create an ideasketch, but when I want to edit, I get an KeyError: 'collaborators' on the fifth form. 
Are there also changes to the ideas models, you didn't commit?

**Comment by slomo** [comment]
Merged master and found the `KeyError`. It occurred only if no invites or collaborators were present. Nicely done @fuzzylogic2000 .

**Review by fuzzylogic2000 — COMMENTED** [review]
- Nice! 
- The link in the invitation email does not work (I can not accept the invitation)
- Create Proposal does not know/show the existing collaborators and invites

**Pull request merged**